tests/rados/test_pool_osd_recovery.py

```diff
@@ -394,6 +396,30 @@ def remove_custom_host(host_node_name: str):
         log.info(
             "---- Starting workflow ----\n---- 5. Removal and addition of OSD daemons"
         )
+
+        # Removing a mon daemon and adding it back post OSD addition.
```

Comment:
Why do we want to remove a mon before OSD removal and add it back after OSD addition? Does it have any special significance?

Reply:
Yes, the mon maps and the OSD maps need to be generated and there would be change in the epochs.